### DISCUSSION TITLE
[Alpha] Fact

### DISCUSSION THREAD

**Discussion by sjur-gr** [navikt/Designsystemet#2405] [Komponenter]
## Velkommen til vurderingen av komponenten `Fact` 🙌

`Fact` er en komponent som viser faktainformasjon eller ekstra informasjon for brukerne. Det er et panel som viser litt av innholdet til å starte med. Bruker kan klikke på en knapp for å se alt innholdet. Samme knapp brukes for å krympe panelet igjen.
![Fact-demo4](https://github.com/navikt/aksel/assets/82579704/af088d46-5fcd-4fe1-b064-2514d3a0d5a1)

Vi har sett et eksempel hvor komponenten er brukt, men er usikre på om det er flere som har behov for den. Eksemplet vi har sett er en situasjon hvor `Fact` blir brukt er under en graf for å vise dataene fra grafen i et annet format.

<img width="796" alt="demo inline" src="https://github.com/navikt/aksel/assets/82579704/9e59a3da-6cc2-4a67-85b3-6a76589ac2e4">

Fra interwebsen vet vi at `Fact` ofte blir brukt for vise kortversjon/oppsummering av en artikkel eller faktaopplysninger om ting som beskrives i artikkelen.

Vi har lagt et eksempel på komponenten [Fact i Alpha UI kit](https://www.figma.com/file/Jw3DCrpJgK0uiG7v92Wdnk/Alpha-UI-kit?type=design&node-id=2147-394&mode=design) som dere kan teste ut.

For å finne ut om vi skal inkludere `Fact` som en komponent i Aksel trenger vi deres hjelp. **Vi er på jakt etter bevis på behov i løsninger som er i prod/demo.** Vi ønsker skjermbilder (uten sensitive opplysninger) og en forklaring av konteksten.

Når vi har fått bevis på minst 3 tilfeller av bruk vil vi gå videre med komponenten. Del dette gjerne med team du tror `Fact` vil passe for.

**Comment by finiteattention** [comment]
Først, disclaimer: jeg har foreløpig ingenting slik i produksjon og dermed ingen aksjer i saken :) Men jeg lurer på hvordan er komponenten tenkt brukt sammenlignet med Accordion eller Readmore? Er det for det meste oppmerksomhetsstyring? Altså: Fact tar ganske mye plass og egner seg kanskje til noe litt _chunkier_ en en forholdsvis enkel forklaring slik jeg kunne tenkt å bruke Readmore til. Accordion derimot kan inneholde ganske _chunky_ innhold. Er forskjellen kanskje at Fact byr på en liten _teaser_ ved å vise et par linjer fra starten? Hilsen, nysgjerrig :)

**Comment by CecilieDy** [comment]
Jeg synes dette er et godt tillegg til Readmore og Accordion, og synes foreløpig at det kan fungere for å vise og skjule resten av lange tabeller som vi trenger. Men vil gjerne høre fra et UU-perspektiv om det er et godt brukstilfelle å bruke Fact til lange tabeller, og om man er nødt til å innlede med en tittel over tabellen (f.eks. "Tabellvisning") eller om man heller kunne vært mer spesifikk i knappen som åpner den, om den kunne f.eks. hete "Vis hele tabellen" i stedet for "Vis mer"? Se eksempel fra skisse til pensjonskalkulator:
![Fact eksempel pensjonskalkulator](https://github.com/navikt/aksel/assets/125269705/621d98bc-a890-4533-8e52-e8f33c57211c)

**Comment by solveiggrieg** [comment]
Vi tenker at vi kan bruke denne komponenten til å skjule lange maler. Innholdet i disse vil saksbehandlerne etterhvert kjenne ganske godt og har ikke behov for at det til enhver tid er åpent. Det gjør det også mer effektivt å scrolle opp til annen informasjon de legger inn i Nærmere begrunnelse for skjønnsfastsettelse.
![fact](https://github.com/navikt/aksel/assets/49483110/6a8154e8-ab4d-4509-a385-1f4ce4555632)

**Comment by CecilieDy** [comment]
Vi utvikler denne Fact-komponenten lokalt hos oss nå i Team pensjonskalkulator, så den kan ev. hentes inn i Aksel eller andre team om ønskelig/mulig.

**Comment by julianacms** [comment]
Vi i team Pensjonsbruker lagde noe lignende i uføretrygd/inntektsplanlegger. Kjempefint hvis teksten på knappen kan tilpasses

<img width="662" alt="Screenshot at May 14 14-38-36" src="https://github.com/user-attachments/assets/0b3b50ad-c9ed-4455-80b6-49a7f9715778" />
